code/Makefile

```diff
@@ -20,18 +20,18 @@ DATA_DIR  = drasil-data
 DOCL_DIR  = drasil-docLang
 EXAM_DIR  = drasil-example
 
-GAME_EXE = chipmunkdocs
 TINY_EXE = tiny
-SWHS_EXE = swhs
+GLASS_EXE= glassbr
 NoPCM_EXE  = nopcm
+SWHS_EXE = swhs
 SSP_EXE  = ssp
-GLASS_EXE= glassbr
+GAME_EXE = chipmunkdocs
 
-DIRS = $(SWHS_DIR) $(NoPCM_DIR) $(TINY_DIR) $(SSP_DIR) $(GLASS_DIR) $(GAME_DIR) $(DATA_DIR) $(LANG_DIR) $(DOCL_DIR) $(EXAM_DIR) $(CODE_DIR) $(GEN_DIR) $(PRINT_DIR)
+DIRS = $(TINY_DIR) $(GLASS_DIR) $(NoPCM_DIR) $(SWHS_DIR) $(SSP_DIR) $(GAME_DIR) $(DATA_DIR) $(LANG_DIR) $(DOCL_DIR) $(EXAM_DIR) $(CODE_DIR) $(GEN_DIR) $(PRINT_DIR)
 DIFF = diff --strip-trailing-cr --ignore-all-space
 
-TESTS = swhs_diff tiny_diff ssp_diff glassbr_diff gamephys_diff nopcm_diff
-PROGS = swhs_prog tiny_prog ssp_prog glassbr_prog gamephys_prog nopcm_prog
+TESTS = tiny_diff glassbr_diff nopcm_diff swhs_diff ssp_diff gamephys_diff
```

Comment:
You only needed to change these two lines, but that's fine.

Reply:
I figured I'd just keep the order of the whole file consistent 😄

Reply:
I thought so, and that's great for readability's sake 👍